src_c/draw.c

```diff
@@ -74,20 +74,20 @@ clip_and_draw_aaline(SDL_Surface *surf, SDL_Rect *rect, Uint32 color,
 static int
 clip_and_draw_line_width(SDL_Surface *surf, SDL_Rect *rect, Uint32 color,
                          int width, int *pts);
-static int
-clipline(int *pts, int left, int top, int right, int bottom);
+//static int
```

Comment:
C usually uses this type of comment.
```
/*
*/
```

I can't remember if `//` is ok in old-C.